pic-api/src/main/kotlin/com/mashup/pic/security/authentication/UserInfo.kt

```diff
@@ -0,0 +1,21 @@
+package com.mashup.pic.security.authentication
+
+import com.mashup.pic.domain.user.User
+import com.mashup.pic.domain.user.UserRole
+
+
+data class UserInfo(
+        val id: Long,
+        val nickname: String,
+        val roles: Set<UserRole>
+) {
+    companion object {
+        fun from(user: User): UserInfo {
+            return UserInfo(
+                    user.id,
+                    user.nickname,
+                    user.roles
```

Comment:
이것도 취향인데 저는 멀티 라인으로 생성자 호출하는 경우 separate line + parameter name을 같이 적어주거덩요? 왜냐면 Human Error 방지차원해서. 이렇게만 보면 잘 들어갔을거라 생각하지만 간혹 순서가 잘못되는경우가 많아서 ( 특히나 field에 변경이 있을 때 좀 잦았어서)
```suggestion
            return UserInfo(
                    id = user.id,
                    nickname = user.nickname,
                    roles = user.roles
```